packages/react-chat/src/components/Form/ChannelMenu.tsx

```diff
@@ -56,112 +57,121 @@ export const ChannelMenu = ({
     [setShowChannelMenu, setShowSideMenu]
   );
 
+  const ref = useRef(null);
+  useClickOutside(ref, () => setShowMenu(false));
+
   if (showMenu || setShowChannelMenu) {
     return (
-      <ChannelDropdown className={className}>
-        {narrow && !className && (
-          <MenuItem
-            onClick={() => {
-              if (switchMemberList) switchMemberList();
-              setShowMenu(false);
-            }}
-          >
-            <MembersSmallIcon width={16} height={16} />
-            <MenuText>{showNarrowMembers ? "Hide" : "View"} Members</MenuText>
-          </MenuItem>
-        )}
-        {channel.type === "group" && (
-          <>
+      <div ref={ref}>
+        <ChannelDropdown className={className}>
```

Comment:
In `DropdownMenu.tsx` we can do
```
type DropdownMenuProps = {
  children: ReactNode;
  className?: string;
  style?: { top: number; left: number };
  ref?: React.RefObject<HTMLHeadingElement>
};

export function DropdownMenu({
  children,
  className,
  style,
  ref
}: DropdownMenuProps) {
  return (
    <MenuBlock className={className} style={style} ref={ref}>
      <MenuList>{children}</MenuList>
    </MenuBlock>
  );
}
```
and then here we can change to 
```suggestion
        <ChannelDropdown className={className} ref={ref}>
```